packages/react/src/components/Dropdown/DropdownSelectedItem.tsx

```diff
@@ -126,13 +129,20 @@ class DropdownSelectedItem extends UIComponent<ReactProps<DropdownSelectedItemPr
     const iconElement = Icon.create(icon, {
       defaultProps: {
         'aria-label': `Remove ${header} from selection.`, // TODO: Extract this in a behaviour.
-        className: DropdownSelectedItem.slotClassNames.removeIcon,
+        className: DropdownSelectedItem.slotClassNames.icon,
       },
       overrideProps: this.handleIconOverrides,
     })
     const imageElement = Image.create(image, {
       defaultProps: {
         avatar: true,
+        className: DropdownSelectedItem.slotClassNames.image,
+      },
+    })
+    const contentElement = Box.create(header, {
+      defaultProps: {
+        as: 'span',
+        className: DropdownSelectedItem.slotClassNames.header,
```

Comment:
Same here, please add `styles.content`, `styles.image` and `styles.icon` to the appropriate slots. The first way of styling we want to advertise is the slot's based. The `classNames` should be used just as an addition to this, not first way of styling slots.